### ISSUE TITLE
Specify Token When creating a user

### ISSUE THREAD

**Issue by d02023287** [influxdata/influxdb#21478]
__Proposal:__
Allow the definition of the token with  influx auth create instead of it being auto generated

__Current behavior:__
Only influx setup -t allows the specification of the token while influx auth create does not allow it

__Desired behavior:__
The ability to define the value of the token when using influx auth create

__Use case:__
Simplifies implementation of multiple InfluxDBs behind a load balancer

**Comment by coxifred** [comment]
Is there any prevision for this option ?
Or simply a commercial way of life to use cluster version ?

**Comment by coxifred** [comment]
Update ?

**Comment by RasmusGodske** [comment]
I agree, this would be a nice feature, especially for infrastructure as code project. Right now the easiest solution is to simply abuse the setup token, which also have admin privileges, which is a huge security concern.

**Comment by jazzmano** [comment]
Agreed, we need this feature to ensure IAC projects
